recipes/xpore/meta.yaml

```diff
@@ -0,0 +1,60 @@
+{% set name = "xpore" %}
+{% set version = "0.5.6" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: 2478c434f40aa24e901fc177a726f2f90fbf09b87ac530cdbc23908729c907be
+
+build:
+  number: 0
+  noarch: generic
+  entry_points:
+    - xpore-dataprep=xpore.scripts.dataprep:main
+    - xpore-diffmod=xpore.scripts.diffmod:main
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  host:
+    - h5py >=2.10.0
```

Comment:
remove everything from the host section that is not, python or pip

Reply:
ok